### PR TITLE
CDPPOC-2 Case MS scaffolding

### PR DESCRIPTION
**Pull request by MEM2677**
Scaffold a Spring Boot MS (taking inspiration from https://github.com/cecchisandrone/entando-codemotion-bundle) with the following capabilities:

Persistence and database provisioning

Keycloak integration

### CODE REVIEW

File: entando.json [hunk 1]
```diff
@@ -3,7 +3,11 @@
         {
             "name": "case-service",
             "stack": "spring-boot",
-            "healthCheckPath": "/api/health"
+            "healthCheckPath": "/api/health",
+						"dbms": "mysql",
```
Comment (cecchisandrone) [inline entando.json]: fix indentation

File: entando.json [hunk 2]
```diff
@@ -3,7 +3,11 @@
...
+            "ingressPath": "/cm",
```
Comment (cecchisandrone) [inline entando.json]: this is not needed, better using canonical one (autogenerated)

File: entando.json [hunk 3]
```diff
@@ -3,7 +3,11 @@
...
+            "deploymentBaseName": "case-management",
```
Comment (cecchisandrone) [inline entando.json]: not needed as well

File: microservices/case-service/src/main/java/org/entando/codemotion/UserServiceApplication.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package org.entando.codemotion;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class UserServiceApplication {
```
Comment (cecchisandrone) [inline microservices/case-service/src/main/java/org/entando/codemotion/UserServiceApplication.java]: Rename this to `CaseServiceApplication`

**Comment by cecchisandrone** [comment]
what's `svc/keycloak.tgz`?

File: microservices/case-service/src/main/java/org/entando/codemotion/UserServiceApplication.java (hunk 4 above)
Comment (MEM2677) [reply microservices/case-service/src/main/java/org/entando/codemotion/UserServiceApplication.java]: done!

File: entando.json (hunk 2 above)
Comment (MEM2677) [reply entando.json]: ingressPath deleted

File: entando.json (hunk 3 above)
Comment (MEM2677) [reply entando.json]: deploymentBaseName deleted

File: entando.json (hunk 1 above)
Comment (MEM2677) [reply entando.json]: done

**Review by cecchisandrone — APPROVED** [review]

**Pull request merged**